src/test/java/uk/gov/digital/ho/hocs/casework/api/factory/strategies/CopyBfToBf2Test.java

```diff
@@ -28,12 +28,12 @@
 
     static {
         FROM_CLOB = Map.of(
-                "CaseSummary", "TestValue");
+                "ComplaintReason1", "TestValue");
```

Comment:
`ComplaintReason1` related to this change?

Also sort of feels like you should have the caseSummary etc in this.

Reply:
Will also resolve what should be in FROM_CLOB